Functions/xASL_fsl_SetFSLdir.m

```diff
@@ -34,7 +34,11 @@
     x = struct;
 end
 if nargin<2 || isempty(bAutomaticallyDetectFSL)
-    bAutomaticallyDetectFSL = false;
+    if isfield(x,'external') && isfield(x.external,'bAutomaticallyDetectFSL')
```

Comment:
This might need the attention of @HenkMutsaerts as well. Not sure what is the best solution - removing the bAutomatically parameter completely from the function call and only using inside the x-struct version... But this version of default vs x-struct vs implicitly passed might not be bad either... TBD